IsTheSumOfLettersEvenOrOdd.rb

```diff
@@ -0,0 +1,52 @@
+=begin 
+Create a function that takes a string and returns true if the sum of the position of every letter in the alphabet is even and false if the sum is odd.
+
+Examples
+
+is_alpha("i'am king")  ➞ true
+# 9 + 1 + 13 + 11 + 9 + 14 + 7 = 64 (even)
+
+is_alpha("True") ➞ true
+# 20 + 18 + 21 + 5= 64 (even)
+
+is_alpha("alexa") ➞ false
+# 1 + 12 + 5 + 24 + 1= 43 (odd)
+Notes
+Case insensitive.
+Ignore non-letter symbols.
+
+Algorithm:
+
+1.We know the alphabets order. like A is 1, B is 2 and goes on.
+2.Now lets take a sentence and from it lets take the first word. We need to find each alphabet's corresponding order number that of word and sum it. Like "in" 9+14 = 23.
+3.Similarly we do the same for the all words in that sentence.
+4.We add all the sums together.
+=end 
+def even_or_odd(hash, string)
+  sum  = 0
+  string.each do |letter|
+    sum += hash[letter] 
+  end
+  if  sum % 2 != 0 
+    "Odd"
+  else 
+    "Even"
+  end
```

Comment:
you are ment to return true or false as part of the question. not odd or even. 

carful with these small things. for online interviews, they would expect you to do what they have asked in the question.

Reply:
Sorry i missed that, will be  careful about it. I had fixed it now

Reply:
can yo push up the new code update for this or?

Reply:
Sure i will

Reply:
To create a new pull request i do have to create a new branch and push the file into it right ?

Reply:
So updates are always done in the same branch. So checkout this branch, make updates and push up. So this branch will be updated. Everything related to this stays in the same branch.

Reply:
Please do correct if i have  wrongly interpreted what you have said. So,   I checkout into this branch, made some updates and pushed it. which is this commit "IsTheSumOfLettersEvenOrOdd.rb changes made!" below. Is this what you meant by the previous comment ?

Reply:
yup this is correct 👍